### PR TITLE
fix: billing address overwrites shipping address when copy flag is set

### PR DESCRIPTION
**Pull request by KevinGorjan**
Fix for #480 

## Description
When addresses are the same, hide the form on the billing step so users can't fill in a new address

## Screenshots (if appropriate):
billing

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/pages/Checkout/Billing.vue [hunk 1]
```diff
@@ -9,23 +9,30 @@
...
-        v-model="setAsDefault"
-        v-e2e="'billing-addresses'"
-        :current-address-id="currentAddressId || NOT_SELECTED_ADDRESS"
-        @setCurrentAddress="handleSetCurrentAddress"
-      />
       <SfCheckbox
         v-e2e="'copy-address'"
         :selected="sameAsShipping"
-        label="Copy address data from shipping"
+        :label="$t('My billing and shipping address are the same')"
```
Comment (Frodigo) [inline packages/theme/pages/Checkout/Billing.vue:15]: Please add a german translation for that to de.js file in the lang directory.

File: packages/theme/pages/Checkout/Billing.vue [hunk 2]
```diff
@@ -243,7 +250,7 @@
             to="localePath('/checkout/shipping')"
             class="sf-button sf-button--underlined form__back-button smartphone-only"
           >
-            Go back
+            {{ $t('Go back') }}
```
Comment (Frodigo) [inline packages/theme/pages/Checkout/Billing.vue:253]: PLease double-check if the translation for that phrase exists in de.js file.

**Review by Frodigo — CHANGES_REQUESTED** [review]

File: packages/theme/pages/Checkout/Billing.vue (hunk 1 above)
Comment (KevinGorjan) [reply packages/theme/pages/Checkout/Billing.vue:15]: Added

File: packages/theme/pages/Checkout/Billing.vue (hunk 2 above)
Comment (KevinGorjan) [reply packages/theme/pages/Checkout/Billing.vue:253]: Added

**Review by Frodigo — APPROVED** [review]

**Pull request merged**